octopusdeploy_framework/schemas/schema.go

```diff
@@ -432,3 +433,14 @@ func GetSensitiveResourceSchema(description string, isRequired bool) resourceSch
 
 	return s
 }
+
+func GetDateTimeResourceSchema(description string, isRequired bool) resourceSchema.Attribute {
+	regex := "^((?:(\\d{4}-\\d{2}-\\d{2})T(\\d{2}:\\d{2}:\\d{2}(?:\\.\\d+)?))(?:Z|[\\+-]\\d{2}:\\d{2})?)"
```

Comment:
this is the [RFC3339](https://www.rfc-editor.org/rfc/rfc3339) specification 
https://regex101.com/r/qH0sU7/1